### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Adda0**
This PR replaces the original Z3 `README.md` with a new one informing about Z3-Noodler, its build system and important links and references.

### CODE REVIEW

**Comment by vhavlena** [comment]
I think it is better to have an email instead of a github page at each author.

**Comment by Adda0** [comment]
That can be done. Both variants are common among GitHub repos.

File: README.md [hunk 1]
```diff
@@ -1,287 +1,137 @@
...
-[3]: #building-z3-using-cmake
-[4]: #z3-bindings
-
-## Building Z3 on Windows using Visual Studio Command Prompt
-
-32-bit builds, start with:
-
-```bash
-python scripts/mk_make.py
+To build tests for Z3-Noodler and run the following
```
Comment (jurajsic) [inline README.md:41]:
```suggestion
To build tests for Z3-Noodle, run the following 
```

**Review by jurajsic — APPROVED** [review]
Looks fine

**Review by vhavlena — APPROVED** [review]

**Comment by Adda0** [comment]
@vhavlena The PR should be done. Merge the PR at your leisure.

**Pull request merged**